.github/workflows/auto_cherry_pick.yml

```diff
@@ -52,6 +53,12 @@ jobs:
         with:
           fetch-depth: 0
 
+      ## Set env var for dependencies label PR
+      - name: Set env var is_dependency to `dependencies` to set the label
+        if: contains(github.event.pull_request.labels.*.name, 'dependencies')
```

Comment:
maybe good to check if the PR author is dependabot, then add the `dependencies` label.

Reply:
Not sure why its so important to check author for dependencies. Anyone with immediate need want to open a PR for bumping the dependency version can also tag PRs with just `dependencies` for automerge !

We otherwise dont tag PRs with `dependencies` label !

Reply:
I was thinking just one more layer of checking, It's non-blocking.